docs/protocol/specifications/validation.md

```diff
@@ -294,55 +131,47 @@ Note: requirement for the input and output sums to be below 2<sup>63</sup> impli
 
 **Algorithm:**
 
-1. Let S be the input blockchain state.
-2. For each transaction in the block:
-    1. [Apply the transaction](#apply-transaction) to S, yielding a new state S′.
-    2. Replace S with S′.
-3. Replace the block header in S with the input block header, yielding a new state S′.
-4. Replace S with S′.
-5. Remove elements of the issuance memory in S where the expiration timestamp is less than the block’s timestamp, yielding a new state S′.
-6. Return S′.
+1. Evaluate the [consensus program](blockchain.md#block-header):
```

Comment:
This is redundant with step 2 (validate the block - which should really be "validate the block header"), which includes evaluation of the consensus program.